src/api/v2/agent-templates/services/posthog.ts

```
@@ -0,0 +1,135 @@
+ * singleton-override pattern used throughout the codebase.
+ */
+
+/* eslint-disable @typescript-eslint/no-require-imports, @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-unsafe-member-access, @typescript-eslint/no-unsafe-call, @typescript-eslint/no-explicit-any */
+
+import type { GenerationMetrics } from "./templateGen";
+
+// ---------------------------------------------------------------------------
+// Event name — single source of truth
+// ---------------------------------------------------------------------------
+
+export const BUILDER_GENERATION_COMPLETED_EVENT = "builder.generation.completed";
```

ESLint

[error] Insert `⏎·`

(prettier/prettier)